<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>学生成绩管理系统</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="studentTable">
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>数学</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>物理</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>英语</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总分</string>
       </property>
      </column>
      <header>
       <property name="sectionResizeMode">
        <enum>QHeaderView::Stretch</enum>
       </property>
      </header>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>添加学生</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>删除学生</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findButton">
        <property name="text">
         <string>查找学生</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sortButton">
        <property name="text">
         <string>按总分排序</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>加载文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>保存文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar"/>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>